README.md

```diff
@@ -48,26 +48,27 @@ CREATE EXTENSION pg_duckdb;
 - `SELECT` queries executed by the DuckDB engine can directly read Postgres tables. (If you only query Postgres tables you need to run `SET duckdb.force_execution TO true`, see the **IMPORTANT** section above for details)
 	- Able to read [data types](https://www.postgresql.org/docs/current/datatype.html) that exist in both Postgres and DuckDB. The following data types are supported: numeric, character, binary, date/time, boolean, uuid, json, and arrays.
 	- If DuckDB cannot support the query for any reason, execution falls back to Postgres.
-- Read parquet and CSV files from object storage (AWS S3, Cloudflare R2, or Google GCS).
-	- `SELECT n FROM read_parquet('s3://bucket/file.parquet') AS (n int)`
-	- `SELECT n FROM read_csv('s3://bucket/file.csv') AS (n int)`
-	- You can pass globs and arrays to these functions, just like in DuckDB
-- Enable the DuckDB Iceberg extension using `SELECT duckdb.install_extension('iceberg')` and read Iceberg files with `iceberg_scan`.
-- Write a query — or an entire table — to parquet in object storage.
-	- `COPY (SELECT foo, bar FROM baz) TO 's3://...'`
-	- `COPY table TO 's3://...'`
-- Read and write to Parquet format in a single query
-
-	```sql
-	COPY (
-	  SELECT count(*), name
-	  FROM read_parquet('s3://bucket/file.parquet') AS (name text)
-	  GROUP BY name
-	  ORDER BY count DESC
-	) TO 's3://bucket/results.parquet';
-	```
-
-- Query and `JOIN` data in object storage with Postgres tables, views, and materialized views.
+- Read and Write support for object storage (AWS S3, Cloudflare R2, or Google GCS):
+	- Read parquet and CSV files:
+		- `SELECT n FROM read_parquet('s3://bucket/file.parquet') AS (n int)`
+		- `SELECT n FROM read_csv('s3://bucket/file.csv') AS (n int)`
+		- You can pass globs and arrays to these functions, just like in DuckDB
+	- Enable the DuckDB Iceberg extension using `SELECT duckdb.install_extension('iceberg')` and read Iceberg files with `iceberg_scan`.
+	- Write a query — or an entire table — to parquet in object storage.
+		- `COPY (SELECT foo, bar FROM baz) TO 's3://...'`
+		- `COPY table TO 's3://...'`
+		- Read and write to Parquet format in a single query
+
+			```sql
+			COPY (
+			SELECT count(*), name
+			FROM read_parquet('s3://bucket/file.parquet') AS (name text)
+			GROUP BY name
+			ORDER BY count DESC
```

Comment:
nit: This part of the query is now missing some indentation that it had before. I think it looked better with the indentation.